src/css/your-order.css

```diff
@@ -0,0 +1,206 @@
+/* ===========YOUR ORDER========= */
+
+.backgroung-img {
+  width: 375px;
+  height: 500px;
+  margin: 0 auto;
+  background-image: url(/img/your-order/green-girl-mobile\ 12.21.16.jpg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  margin-bottom: 80px;
+}
+
+@media screen and (min-resolution: 2dppx), (min-resolution: 192dpi) {
+  .backgroung-img {
+    background-image: url(/img/your-order/green-girl-mobile@2x\ 12.21.16.jpg);
+  }
```

Comment:
@media screen and (min-resolution: 2dppx), (min-resolution: 192dpi) {
  .backgroung-img {
    background-image: url(/img/your-order/green-girl-mobile@2x.jpg);
  }